controller/search.go

```diff
@@ -33,14 +34,18 @@ func NewSearchController(service *goa.Service, db application.DB, configuration
 // Users runs the user search action.
 func (c *SearchController) Users(ctx *app.UsersSearchContext) error {
 
+	_, err := login.LoadContextIdentityIfNotDeprovisioned(ctx, c.db)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError(err.Error()))
```

Comment:
Don't have to wrap it in errors.NewUnauthorizedError(). LoadContextIdentityIfNotDeprovisioned() will return NewUnauthorizedError.

Reply:
But the controller should be explicitly making it a 401 irrespective of what is returned from there ?

Reply:
Sure. But "return jsonapi.JSONErrorResponse(ctx, err)" is enough. err is already NewUnauthorizedError.

Reply:
Anyway. It's a minor thing. Merging.